app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show; end
+
+  def new; end
+
+  def create; end
+
+  def edit; end
+
+  def update; end
+
+  def destroy; end
```

Comment:
If this PR is supposed to include only the `index` action, the other methods sould not exist.